apigee/adapter.go

```diff
@@ -211,6 +212,9 @@ func (*builder) SetQuotaTypes(map[string]*quotaT.Type)         {}
 func (h *handler) Close() error {
 	h.productMan.Close()
 	h.authMan.Close()
+	if h.analyticsMan != nil {
+		h.analyticsMan.Close()
```

Comment:
This could be done as nil check in analyticsMan like is done in auth.Close()

Reply:
The problem is analyticsMan is an interface, the other ones are structs. I believe if you do the checks in the implementing structs you run into nil typing issues:
https://stackoverflow.com/questions/19761393/why-does-go-have-typed-nil

Reply:
ok. no worries.